UPDATING.md

```diff
@@ -14,3 +14,4 @@
 binaries need to be rebuilt with the proper offset using `uf2tool`.
 - On ESP32, SSID and PSK stored in NVS are no longer read by network module. Applications
 must fetch the values and pass them to `network:start/1` or `network:start_link/1`.
+- The `lib.avm` partition is no longer supported on ESP32.
```

Comment:
Sorry for the nitpicking, the pattern I would like to keep is "X happened, you should do Y for updating".

So something might be an idea: "The `lib.avm` partition is no longer supported on ESP32. If you are using it ... you should use released images or update partitioning schema and use a boot image" or something like that.